mypy/errorcodes.py

```diff
@@ -34,6 +34,9 @@ def __init__(
             sub_code_map[sub_code_of.code].add(code)
         error_codes[code] = self
 
+    def is_import_related_code(self) -> bool:
+        return IMPORT in (self.code, self.sub_code_of)
```

Comment:
(Informtional:) This could easily be generalized to `is_related_code`, but hey: he who actually wants that feature is free to do that refactor, YAGNI, etc.